### PR TITLE
UIDATIMP-489 Field Mapping Profile details: MARC Bib from MARC Bib. Edit

### PR DESCRIPTION
**Pull request by IvanKryzhanovskyi**
## Description
Add validation to form fields for the Edit action.

## Approach
1. Add validation functions to appropriate fields
2. Add tests

## Tickets
[UIDATIMP-489](https://issues.folio.org/browse/UIDATIMP-489)
[UIDATIMP-490](https://issues.folio.org/browse/UIDATIMP-490)
[UIDATIMP-491](https://issues.folio.org/browse/UIDATIMP-491)

### CODE REVIEW

File: src/components/MARCTable/MARCTableRow.js [hunk 1]
```diff
@@ -111,69 +111,49 @@ export const MARCTableRow = ({
...
-      if (actionValue === DELETE) {
-        return validateRequiredField(value) || validateAlphanumericOrAllowedValue(value, '*') || validateSubfieldField(fieldValue)(value);
-      }
-
-      if (actionValue === MOVE) {
-        return validateAlphanumericOrAllowedValue(value, '*');
-      }
-
-      return null;
+      return validateRequiredField(value) || validateAlphanumericOrAllowedValue(value, '*') || validateSubfieldField(fieldValue)(value);
```
Comment (mariia-aloshyna) [inline src/components/MARCTable/MARCTableRow.js:144]: Actually I'm not sure that Subfield is required for the Move action.

Comment (IvanKryzhanovskyi) [reply src/components/MARCTable/MARCTableRow.js:144]: Looks like it is. UIDATIMP-492, Scenario 6: Values other than *, alpha, numeric in Subfield column 6

Comment (mariia-aloshyna) [reply src/components/MARCTable/MARCTableRow.js:144]: Hmm, at the same time at screenshots there is no value for it.

Comment (mariia-aloshyna) [reply src/components/MARCTable/MARCTableRow.js:144]: Ok, let's stick to the description.

Comment (IvanKryzhanovskyi) [reply src/components/MARCTable/MARCTableRow.js:144]: Hmm, probably you are right, i'll leave subfield field as optional for MOVE action

**Pull request merged**